test/e2e/lifecycle/reboot.go

```diff
@@ -59,6 +59,9 @@ var _ = SIGDescribe("Reboot [Disruptive] [Feature:Reboot]", func() {
 
 		// Cluster must support node reboot
 		framework.SkipUnlessProviderIs(framework.ProvidersWithSSH...)
```

Comment:
I think this already skips "skeleton" 🤔
We have other local clusters that set skeleton instead of local for conformance testing &/or avoiding some of the baggage local has with invoking the scripts in cluster/

Cc @dims